.gitignore

```diff
@@ -4,3 +4,4 @@
 
 .vagrant
 *.sublime-project
+*_stringer.go
```

Comment:
please check in op_stringer.go instead of excluding the file.

please don't use go generate on Travis either.